my-app/src/CreateItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
The component names are always nouns and can never be verbs.

Functions are verbs as they perform some action. Components are entities that are rendered on your page.

Rename this component to be called `Item`.

Also, correct this naming in all the components